index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Bhekisile's Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <div class="container">
+        <nav class="nav_bar">
+                <span class="start-nav">My Logo</span>
+                <span class="end-nav"><a href="#">Contact</a></span>
```

Comment:
I love the color of your logo 👌but here is what you can improve on the navbar:
1. Please make sure to use the proper font size and font weight for all of your texts, including the log. Because it's required to follow the original Figma template.
2. It's important to put your logo in an `a` tag so that when the user clicks on the logo it'll return it to the home page.
3. As you can see in the original design on the right side it's a humbergar button, and you can replace your `contact` link with a `humbergar` icon. Please make sure to update this.

Your design: 

![image](https://user-images.githubusercontent.com/34779928/205122343-a4f25463-d853-4654-8520-78f05e9a6c4e.png)


Original Design:

![image](https://user-images.githubusercontent.com/34779928/205122392-63fa590b-a1b8-49f1-9fad-1a82aa3a0bae.png)